modules/cassandra/src/main/scala/sup/cassandra.scala

```diff
@@ -0,0 +1,16 @@
+package sup
+
+import cats.Id
+import cats.effect.kernel.{Async, Sync}
+import com.datastax.oss.driver.api.core.CqlSession
+
+import scala.concurrent.duration.FiniteDuration
+
+object cassandra {
+
+  def connectionCheck[F[_] : Async](session: CqlSession, timeout: FiniteDuration): HealthCheck[F, Id] =
+    HealthCheck.liftFBoolean {
+      Async[F].timeoutTo(Sync[F].delay(session
```

Comment:
The same questions / suggestions as in #362, in this case the blocking will have to be done with `Sync[F].interruptible` so the blocking action can be interrupted in case of timeout (I think `many = true` as the parameter is reasonable).

Reply:
I rewrote it.